geometry_msgs/Pose goal
---
string status